src/components/SQForm/SQFormDropdown.js

```diff
@@ -138,7 +138,11 @@ SQFormDropdown.propTypes = {
   children: PropTypes.arrayOf(
     PropTypes.shape({
       label: PropTypes.string,
```

Comment:
`value` and `label` should be required here.

We reuse this propType in so many places, I almost wish we had it defined as a constant somewhere.